.circleci/config.yml

```
@@ -297,6 +297,25 @@ jobs:
           paths:
             - ~/mne_data/eeg_matchingpennies
 
+  cache_MNE-funloc-data:
+    <<: *imageconfig
+    steps:
+      - attach_workspace:
+          at: ~/
+      - restore_cache:
+          keys:
+            - data-cache-MNE-funloc-data-1
```

Should match the other one so
```suggestion
            - data-cache-MNE-funloc-data-2
```